collect.py

```diff
@@ -115,6 +127,24 @@ def do_something(driver, elem_id=None):
             elem.click()
         elif input_type == 'number':
             elem.send_keys('3')
+        elif input_type == 'search':
+            try:
+                elem.send_keys('marco search')
+                elem.submit()
+            except BaseException:
+                try:
+                    element = driver.find_element_by_name('search')
```

Comment:
I'm not sure we can rely on the name of the element, here and the following. `q` for example is a pretty generic name, how can we be sure that it is always a search field in any website?